doc/conf.py

```diff
@@ -133,12 +133,20 @@
 
 SUPPORTED_CKAN_VERSIONS = 2
 
+def extract_version(tag):
+    """ Ensure we only work with 'valid' ckan tagged releases"""
+    _regex = re.compile(r"^\s*ckan-" + VERSION_PATTERN + r"\s*$", re.VERBOSE | re.IGNORECASE)
+    match = re.search(_regex, tag)
+    if not match:
+        return False
+    else:
+        return True
 
 def get_release_tags():
     git_tags = subprocess.check_output(
         ['git', 'tag', '-l'], stderr=subprocess.STDOUT).decode('utf8')
     git_tags = git_tags.split()
-    release_tags_ = [tag for tag in git_tags if tag.startswith('ckan-')]
+    release_tags_ = [tag for tag in git_tags if extract_version(tag)]
```

Comment:
This was done to fix forks which make have version data that does not follow the normal pattern. This will remove it from being in the list causing docs to not be built.